index.html

```diff
@@ -24,16 +24,19 @@
         <ul class="navigation-list navigation-user">
           <li class="navigation-item">
             <a class="navigation-link" href="#">
-              <span class="visually-hidden">
+              <img src="images/search.svg" alt="Поиск">
+              <span class="visually-hidden">                
                 Поиск
               </span>
             </a>
           </li>
           <li class="navigation-item">
             <a class="navigation-link" href="#">
+              <img src="images/favorite.svg" alt="Избранное">
               <span class="visually-hidden">
                 Избранное
               </span>
+              <sup class="counter">12</sup>    <!--Очень раздражает наличие счётчика в макете, и его отсутствие в вёрстке :)-->
```

Comment:
тут имеет смысл счётчик вынести в вёрстку, но лучше тогда написать другое скрытое собщние
```suggestion
              <span class="visually-hidden">
                Выбрано гостиниц:
              </span>
              <sup class="counter">12</sup>  
```